src/components/ui/VideoGrid/Styled.tsx

```diff
@@ -218,6 +218,134 @@ const ratioStyles = {
   '17.r1by1.featured': `grid-template-areas:
       'ft ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft ft v'
       'ft ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft ft x';`,
+
+  '18': 'grid-template: repeat(5,1fr) / repeat(4,1fr);',
+  '18.slim': 'grid-template: repeat(8,1fr) / repeat(3,1fr);',
+  '18.r1by2': 'grid-template: repeat(6,1fr) / repeat(4,1fr);',
+  '18.r1by1': 'grid-template: repeat(4,1fr) / repeat(5,1fr);',
+
+  '18.featured': `grid-template: repeat(8,1fr) / repeat(9,1fr);
+    grid-template-areas:
+      'ft ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft ft'
+      'ft ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft ft';`,
+  '18.r1by1.featured': `grid-template: repeat(9,1fr) / repeat(9,1fr);
```

Comment:
Do we need to redefine grid template here `grid-template: repeat(9,1fr) / repeat(9,1fr); ? From the example above, doesn't seem we need to